site-src/guides/getting-started/migrating-from-ingress-nginx.md

```diff
@@ -0,0 +1,59 @@
+# A Welcome Guide for Ingress-NGINX Users
+
+Welcome! If you're an Ingress-NGINX user, you're in the right place. This page is a central hub of resources for those considering or actively migrating to Gateway API. We aim to continue to build out this page to be more comprehensive over time. If you're looking for a general overview of migrating from Ingress, please refer to our [Migrating from Ingress guide](./migrating-from-ingress.md).
+
+We understand that migrations can be complex, and our goal is to provide you with the information and tools you need for a smooth transition.
+
+## Common Questions
+
+**How does Gateway API compare to Ingress?**
+Gateway API provides a role-oriented and more expressive API than Ingress. Where Ingress combines the concepts of a load balancer and routing rules into a single resource, Gateway API splits them apart:
+
+*   **Gateway:** Defines where and how traffic enters the cluster, a task for a cluster operator.
+*   **HTTPRoute:** Defines how traffic is routed to services, a task for an application developer.
+
+This separation allows for safer, multi-tenant infrastructure. For a deeper dive, check the [Ingress Migration Guide](./migrating-from-ingress.md).
+
+**How do I map Ingress-NGINX features to Gateway API?**
+Many annotation-based features in Ingress-NGINX have corresponding fields in Gateway API. For example, traffic splitting, header manipulation, and TLS configuration are all native to the API. For a detailed mapping, we recommend exploring the [Gateway API HTTPRoute documentation](../../api-types/httproute/) and the documentation of your chosen [implementation](https://gateway-api.sigs.k8s.io/implementations/).
```

Comment:
do you want to make also "implementations" a relative link? (../implementations)

Reply:
Updated, thanks!